english/you-are-in-minecraft/minecraft.md

```diff
@@ -0,0 +1,17 @@
+You open your eyes and there you are surrounded by a crafting table, a furnace, and what seems to be a new addition to minecraft, marshmallow walls.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.